website/content/en/docs/building-operators/ansible/tutorial.md

```diff
@@ -102,7 +102,11 @@ This memcached role will:
 - Ensure a memcached Deployment exists
 - Set the Deployment size
 
-It is good practice to set default values for variables used in Ansible
+Note that the tasks in this Ansible role file are what actually defines the behavior of the spec and status of the memcached custom resource.
+As Kubernetes allows entry of arbitrary fields when creating resources, we don't need to actually create specific fields in the CRD. 
+While we won't be doing this in this tutorial, in recommended to also define these fields in the CRD, so that Kubernetes users
```

Comment:
```suggestion
While we won't be doing this in this tutorial, its is recommended to also define these fields in the CRD, so that Kubernetes users
```

Reply:
fixed